lib/matplotlib/tests/test_rcparams.py

```diff
@@ -204,6 +204,14 @@ def test_Issue_1713(tmpdir):
     assert rc.get('timezone') == 'UTC'
 
 
+def test_Issue_17908():
```

Comment:
```suggestion
def test_animation_frame_format():
```
or similar. The function name should contain semantic information on what it tests. It should not be an issue id.

Reply:
Sure, I can do that.  I was patterning off some test names earlier in the module.

While you're looking at it: is it okay to implement the test as I've done here?  Seems like it would leak state across tests.  I don't anticipate this breaking anything, but it seems like bad style to do this without an explicit reset.  I only wrote it this way because it matched the style of other tests in this module.

Reply:
All `rcParams` are reset for every test; there would be no state leakage.